src/components/Scheduling/ConfirmationStyles.js

```diff
@@ -46,7 +46,7 @@ const useStyles = makeStyles((theme) => ({
     alignItems: 'center',
     marginRight: '1rem',
   },
-  day: {
+  dayStyle: {
```

Comment:
I renamed this becuase I had a naming conflict when changing the import from `classes = useStyle()' to `{ day } = useStyle()'